docs/tutorials/credentials-status-update/README.md

```diff
@@ -0,0 +1,155 @@
+# Credentials Status Update Tutorial
+
+If you wish to bypass this tutorial and simply want to import the postman collection locally, copy the "Raw" [link](https://raw.githubusercontent.com/w3c-ccg/traceability-interop/main/docs/tutorials/credentials-status-update/credentials-status-update.postman_collection.json) to your clipboard and In Postman, Import -> Link -> Paste the "Raw" link. Otherwise, please continue with the rest of the tutorial.
```

Comment:
```suggestion
If you wish to bypass this tutorial and simply want to import the postman collection locally, copy the "Raw" [link](https://raw.githubusercontent.com/w3c-ccg/traceability-interop/main/docs/tutorials/credentials-status-update/credentials-status-update.postman_collection.json) to your clipboard. Then, in Postman, **Import** -> **Link** -> **Paste** the "Raw" link. Otherwise, please continue with the rest of the tutorial.
```